src/main/java/de/embl/cba/mobie/n5/zarr/XmlN5OmeZarrImageLoader.java

```diff
@@ -27,6 +39,40 @@ public N5OMEZarrImageLoader fromXml(Element elem, File basePath, AbstractSequenc
         return null;
     }
 
+//    private SpimData openBdvZarrData(String path) {
```

Comment:
Why is this function commented out?

Reply:
it is WIP refactoring to make MoBIE code easier to read.